### ISSUE TITLE
[Accessibility][Screen Reader] Voiceover does not notify the user about the auto suggestions in PeoplePicker on Mac

### ISSUE THREAD

**Issue by nawadih** [microsoft/fluentui#18856]
### Environment Information

- **Package version(s)**:     "@fluentui/react": "^7.129.0",
Device: MacBook Air
MAC OS: 11.3 Big Sur
Browser: Safari 14.0.3 (16610.4.3.1.7)
Screen Reader: Voiceover

### Describe the issue:

Screen reader does not announce the autosuggested list of names that appears when using people picker on Mac using voiceover. On windows this works correctly where the screen reader announces the autosuggestions when navigating through them

### Please provide a reproduction of the issue in a codepen:
This repros in the fluent ui people picker sample: https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker
Repro steps:
1. on a macbook open up the above link and turn on voiceover. 
2. tab into the people picker box, and observe the list appears. 
3. navigate with up and down arrows through the list and you will see none of the suggestions are announced by the screen reader. If you try these same steps on windows + nvda the names will be announced
![image](https://user-images.githubusercontent.com/49041826/124844904-6204fc80-df4a-11eb-87f0-220f85f6b8de.png)

#### Actual behavior:

Screen reader will not announce suggestions in people picker list when navigating up and down with arrow keys.

#### Expected behavior:

Screen reader should read list items, as it does on other platforms

### Documentation describing expected behavior
Our accessibility team filed this as a part of a test pass, this is documented by the Microsoft Accessibility Standard here: 
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.02.12#Zhome. The screen reader should announce the autocomplete suggestions in the people picker

**Comment by ling1726** [comment]
@smhigley  could you see who on the v8 team could have time to look into this ? I can't even take a look since I unfortunately do not own a man 😅

**Comment by smhigley** [comment]
This is a longstanding VoiceOver bug -- you can see the same issue here: https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html

I'm not sure what to do about this since there aren't any clear workarounds. I can file another bug, but there isn't much Fluent can do in the meantime. On the plus side, since PeoplePicker autocompletes, if the user types VoiceOver will read the autocompleted value.

**Comment by ling1726** [comment]
@smhigley  is there a voiceover bug you can link here ?
